Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -275,3 +256,33 @@ you can export the settings from this cluster with the following steps.
 
 !!! important
     For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.
+
+## Upgrades
+
+1. If consumer cluster doesn't have restricted caps, this will upgrade all the default csi-users (non-restricted):
+
+    ```console
+    python3 create-external-cluster-resources.py --upgrade
+    ```
+
+2. If the consumer cluster has restricted caps:
+Restricted users created using `--restricted-auth-permission` flag need to pass mandatory flags: '`--rbd-data-pool-name`(if it is a rbd user), `--k8s-cluster-name` and `--run-as-user`' flags while upgrading, in case of cephfs users if you have passed `--cephfs-filesystem-name` flag while creating csi-users then while upgrading it will be mandatory too. In this example the user would be `client.csi-rbd-node-rookstorage-replicapool` (following the pattern `csi-user-clusterName-poolName`)
+
+    ```console
+    python3 create-external-cluster-resources.py --upgrade --rbd-data-pool-name replicapool --k8s-cluster-name rookstorage --run-as-user client.csi-rbd-node-rookstorage-replicapool
+    ```
+
+!!! note
+    An existing non-restricted user cannot be converted to a restricted user by upgrading.
+    The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
+
+### Upgrade cluster to utilize new feature
+
+Some Rook upgrades may require re-running the import steps, or may introduce new external cluster features that can be most easily enabled by re-running the import steps.
+
+To re-run the import steps with new options, the python script should be re-run using the same configuration options that were used for past invocations, plus the configurations that are being added or modified.
+
+Starting with Rook v1.15, the script stores the configuration in the external-cluster-user-command configmap for easy future reference.
+
+!!! warning
+    Failure to reuse the same configuration options when re-invoking the python script can result in unexpected changes when re-running the import script.
```

Comment:
Shouldn't this be the intro text to `## Upgrades`?

Reply:
No this is only for new features with upgrades